src/identity/client.rs

```diff
@@ -118,6 +136,10 @@ impl PublicId {
     pub fn public_key(&self) -> &PublicKey {
         &self.public_key
     }
+
+    pub fn new(name: XorName, public_key: PublicKey) -> Self {
```

Comment:
Is this something needed only for testing?  Maybe we should feature gate it?

Reply:
I needed it for the `FullId` constructors. Usage is [here](https://github.com/maidsafe/safe-nd/pull/36/files/b5c72383ce2b746280bcda0a884144e77f70d3d5#diff-c0b1a46ed40f1a21040e004922465426R69)